WS2012R2/lisa/setupscripts/NET_filecopy_scp.ps1

```diff
@@ -269,44 +280,44 @@ $localChksum2 = compute_local_md5 $filePath2
 $Error.Clear()
 $command = "${rootDir}\bin\pscp -i ${rootDir}\ssh\${sshKey} ${filePath2} root@${ipv4}:/mnt/"
 $job = Start-Job -ScriptBlock  {Invoke-Expression $args[0]} -ArgumentList $command
-<#$job = Start-Job -ScriptBlock {$rootDir\bin\pscp -i $rootDir\ssh\${sshKey} ${filePath2} root@${ipv4}:/mnt/}#>
 $copyDuration1 = (Measure-Command { bin\pscp -i ssh\${sshKey} ${filePath1} root@${ipv4}:/mnt/ }).totalseconds
 
-while ($True){
+while ($True){f
     if ($job.state -eq "Completed"){
             $copyDuration2 = ($job.PSEndTime - $job.PSBeginTime).seconds
             Remove-Job -id $job.id
        break
     }
 }
 
-
 if ($Error.Count -eq 0) {
     Write-Output "Info: File has been successfully copied to guest VM '${vmName}'" | Tee-Object -Append -file $summaryLog
 }
 else {
-    Write-Output "ERROR: File could not be copied!" | Tee-Object -Append -file $summaryLog
-    $retVal = $False
+    Write-Output "ERROR: An error occured while copying files!" | Tee-Object -Append -file $summaryLog
+    remove_files
+    return $False
 }
 
 [int]$copyDuration = [math]::floor($copyDuration)
 
-Write-Output "The file copy process took ${copyDuration1} and ${copyDuration2} seconds" | Tee-Object -Append -file $summaryLog
+Write-Output "The file copy process took ${copyDuration1} seconds and ${copyDuration2} seconds" | Tee-Object -Append -file $summaryLog
```

Comment:
* should be files
please handle the messages better, to refer that for the 1st file it took an X amount of seconds, and for the 2nd file Y seconds.